.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           echo "##vso[task.setvariable variable=gitSha;isOutput=true]$HEAD_SHA"
           until [ $RETRIES -eq 0 ] || [[ `az acr repository show-tags -n ccfmsrc --repository ccf/ci` == *"$HEAD_SHA"* ]]
           do
-            sleep 60
+            sleep 10
```

Comment:
Ken asked us recently to change this to 60, he thought only waiting 10 seconds wasn't too far from DoS'ing ACR